pavelib/servers.py

```diff
@@ -27,10 +31,12 @@ def run_server(system, settings=None, port=None, skip_assets=False, contracts=Fa
 
     if not settings:
         settings = DEFAULT_SETTINGS
+        asset_settings = settings
```

Comment:
Is this line correct? Why are we setting asset_setting to a DEFAULT_SETTINGS, and only in the case when no settings are provided? Wouldn't it make more sense to leave asset_settings undefined?